.github/workflows/unit-tests-merge-pr.yml

```diff
@@ -1,13 +1,13 @@
 name: Unit Tests using Jest
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches: [main]
+    push:
+        branches:
+            - main
+    pull_request:
+        branches: [main]
 jobs:
-  unit_tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - run: npm ci && npm test
\ No newline at end of file
+    unit_tests:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+            - run: cd app && npm ci && npm test
```

Comment:
I suspect we may have some battling code formatting tools going on. Nothing to change immediately, but worth paying attention to.